Implemented new fields in item_db (#6367)
rathena/rathena: doc/item_db.txt

---------------------------------------

MagicDefense: Magic Armor's defense.

---------------------------------------

Str: Strength bonus given by the equipment. This field is working for Armor, Weapon, and ShadowGear Type.

---------------------------------------

Agi: Agility bonus given by the equipment. This field is working for Armor, Weapon, and ShadowGear Type.

---------------------------------------

Vit: Vitality bonus given by the equipment. This field is working for Armor, Weapon, and ShadowGear Type.

---------------------------------------

Int: Intelligence bonus given by the equipment. This field is working for Armor, Weapon, and ShadowGear Type.

---------------------------------------

> Dex: Dexterity bonus given by the equipment. This field is working for Armor, Weapon, and ShadowGear Type.

Reviewer: ```suggestion
Dex: Dexterity bonus given by equipment. This field works with Armor, Weapon, and ShadowGear Type. This bonus is not given in restricted maps.
```

---------------------------------------

> Luk: Luck bonus given by the equipment. This field is working for Armor, Weapon, and ShadowGear Type.

Reviewer: ```suggestion
Luk: Luck bonus given by equipment. This field works with Armor, Weapon, and ShadowGear Type. This bonus is not given in restricted maps.
```

---------------------------------------

> MaxHp: Maximum HP bonus given by the equipment. This field is working for Armor, Weapon, and ShadowGear Type.

Reviewer: ```suggestion
MaxHp: Maximum HP bonus given by equipment. This field works with Armor, Weapon, and ShadowGear Type. This bonus is not given in restricted maps.
```

---------------------------------------

> MaxSp: Maximum SP bonus given by the equipment. This field is working for Armor, Weapon, and ShadowGear Type.

Reviewer: ```suggestion
MaxSp: Maximum SP bonus given by equipment. This field works with Armor, Weapon, and ShadowGear Type. This bonus is not given in restricted maps.
```

---------------------------------------

> WeaponElement: Element of Weapon/ammo item Type. The field can't be used with others Type of item. The element level is 1.

Reviewer: ```suggestion
WeaponElement: Element of Weapon or Ammo item Type only. The element level is 1.
```

---------------------------------------

> Indestructible: Defines if the equipment can not be broken.

Reviewer: ```suggestion
Indestructible: Defines if the equipment can be broken or not.
```
